status.json

```diff
@@ -6000,5 +6000,12 @@
     "wpd": "",
     "id": 6170807885627392,
     "uservoiceid": null
+  },
+  {
+    "name": "Media Session",
+    "category": "Multimedia",
+    "link": "https://wicg.github.io/mediasession",
+    "summary": "Enable websites to customize media metadata so they can control the notification and lock screen UI. Also, allow them to control media in the page from outside the page, such as platform UIs and media keys.",
+    "standardStatus": "Editor's draft"
```

Comment:
@beaufortfrancois Can you add [`"id": 5639924124483584`](https://www.chromestatus.com/feature/5639924124483584). Thanks!